OpenEdXMobile/res/layout/row_course_outline_list.xml

```diff
@@ -93,15 +91,15 @@
                     tools:visibility="visible" />
 
                 <TextView
-                    android:id="@+id/row_subtitle_due_date"
+                    android:id="@+id/row_subtitle_video_size"
```

Comment:
@bilalawan321 there is a view id change. please update it on your end. Maybe this is also the reason Junkin's build failed.